utils/filter/filter.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2022 Red Hat, Inc.
+// All Rights Reserved.
+//
+//    Licensed under the Apache License, Version 2.0 (the "License"); you may
+//    not use this file except in compliance with the License. You may obtain
+//    a copy of the License at
+//
+//         http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+//    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+//    License for the specific language governing permissions and limitations
+//    under the License.
+
+package filter
+
+import (
+	"strings"
+
+	"github.com/redhatcre/syncron/pkg/cli"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+)
+
```

Comment:
Maybe it would be good to add the explanation of this package? Something similar to what you added in the PR.

Reply:
I mean: It filters, but, why `Component`?

Reply:
Good point @adrianfusco !